.github/workflows/dev_deploy.yml

```
@@ -8,98 +8,85 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    # if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
 
     steps:
+      # 소스코드 체크아웃
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)